test/src/test/testtool/Validators/Schema/openapi-json.json

```diff
@@ -2,13 +2,12 @@
     "openapi":"3.0.0",
     "info":{
         "title":"Ibis4Test",
-        "description":"OpenApi auto-generated at IGNORE-TIMESTAMP for Ibis4Test (IGNORE-STAGE)",
+        "description":"OpenApi auto-generated at 2022-05-11 08:35:44 for Ibis4Test (LOC)",
```

Comment:
Any idea where this IGNORE-TIMESTAMP is coming from? Tests seem to pass so I doubt this will matter much, but it's strange :)

Reply:
it must be ignore content thingy

Reply:
So will you revert the change? I don't understand why the tests aren't failing if it must be ignored?